src/Payment.ts

```diff
@@ -95,12 +95,17 @@ export class Payment {
     networkAddress: string,
     receiverAddress: string,
     value: number | string,
-    options: PaymentOptions = {}
+    options: PaymentOptions = {},
+    referenceId: string = ''
```

Comment:
I think it should be part of the options

Reply:
I also think it could better be named `receiverReference` since `referenceId` could also mean the reference we will use to bound a message to a payment.

Reply:
The reason I did not included into the options was that the `PaymentOptions` are used at multiple places e.g. prepare to close a trustline. Do you think it still fits into that interface or would it require a new interface extending the `PaymentOptions`? :thinking:

Reply:
as we can not come up with a better name, I suggest we stick to `paymentRequestId`

Reply:
It has to go in the options. If it does not fit the type, change the type/create a new one.

Reply:
:heavy_check_mark: